README.md

```diff
@@ -6,44 +6,9 @@
 [![Code Coverage Badge](https://github.com/yardinternet/wp-hook-registrar/blob/badges/coverage.svg)](https://github.com/yardinternet/wp-hook-registrar/actions/workflows/badges.yml)
 [![Lines of Code Badge](https://github.com/yardinternet/wp-hook-registrar/blob/badges/lines-of-code.svg)](https://github.com/yardinternet/wp-hook-registrar/actions/workflows/badges.yml)
 
-An Acorn package for WordPress Hook Registration.
+Register Hooks using php attributes.
 
-## Features
-
-- [x] Register Hooks using php attributes
-- [x] Configure Hook registration using a config file
-- [x] Load plugin-specific hooks only when the plugin is active
-
-See [config](./config/hooks.php) for all configuration options.
-
-## Requirements
-
-- [Sage](https://github.com/roots/sage) >= 10.0
-- [Acorn](https://github.com/roots/acorn) >= 4.0
-
-## Installation
-
-1. Install this package with Composer:
```

Comment:
Installeren is toch nog wel nodig, als je deze package zonder brave hooks gebruikt?